controllers/apply/awards-for-all/confirmation.test.js

```diff
@@ -11,8 +15,8 @@ describe('confirmation builder', () => {
         });
 
         expect(england.body).toMatchSnapshot();
-        expect(england.body).toContain('0345 4 10 20 30');
-        expect(england.body).toContain('afe@tnlcommunityfund.org.uk');
+        expect(england.body).toContain(CONTACT_DETAILS_PHONE.england);
```

Comment:
I'd avoid importing the constants for this test, it's good if unit tests avoid importing implementation details of the code they are testing. We could use a snapshot test here if we want to avoid duplicating values. (I know we do this elsewhere but it's good to avoid)